tests/Unit/PointwiseFunctions/AnalyticData/GrMhd/BondiHoyleAccretion.py

```diff
@@ -0,0 +1,92 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import numpy as np
+
+import PointwiseFunctions.GeneralRelativity.KerrSchildCoords as ks_coords
```

Comment:
This is how we can import functions across test modules now.

This requires `pypp::SetupLocalPythonEnvironment local_python_env{""}` in the cpp test. If no cross-imports are needed, the setup can be done in a submodule.

Reply:
I like this, I think it's a good approach :+1: